# new exception classes for potential catching

class ConstantError(Exception):
    pass
